### ISSUE TITLE
Milestones : Link title can be same name

### ISSUE THREAD

**Issue by deepakanbarasan1** [appirio-tech/connect-app#2328]
**Expected behavior :** 
Describe : Milestones  Link title name should not be same

**Actual behavior**
Describe: Milestones link title accepting same name

Test Feature : Wiring  component for checkpoint-review (Adding more link)

**Description :**

Milestones  Link title name should not be same. It's actual behavior for this field ??

**Project Name :** Test project for V 2.4.4 Web UI Phase timeline - Front end test
**Phase :** Website Development
**Milestone :** community-review

**Web UI :** 
![milestones link title can be same name](https://user-images.githubusercontent.com/30447858/43855469-50261784-9b63-11e8-8f1b-1691d4477065.PNG)

**Comment by deepakanbarasan1** [comment]
@RishiRajSahu Please check

**Comment by vic-topcoder** [comment]
I think this is ok to not implement name check, but that's good thinking on having multiple names. Right now the system automatically adds increments when adding links of the same type.

**Comment by vikasrohit** [comment]
I don't think it something critical for this release, moving to the next release.

**Comment by KevinLeigh** [comment]
If this cannot be completed, please can this issue be assigned to me?

**Comment by Gauravseta** [comment]
@kevinleigh i will complete within few hrs. near to done..

**Comment by KevinLeigh** [comment]
@Gauravseta Thank you for the update, good luck.

**Comment by Gauravseta** [comment]
@gondzo PR raised. #2423

**Comment by gondzo** [comment]
removing from milestone, will fix this internally

**Comment by RishiRajSahu** [comment]
@gondzo do you have the changes for this to be pushed to dev ?
